src/valid/interface.rs

```diff
@@ -369,7 +370,6 @@ impl VaryingContext<'_> {
                                     #[cfg(not(feature = "validate"))]
                                     let _ = index;
                                 }
-                                // TODO: shouldn't this be validate?
```

Comment:
I think this TODO meant, shouldn't this be conditional on the `"validate"` feature. I think we shouldn't be calling this function at all when `"validate"` isn't enabled. But that's not something for this PR; filed as #2302.

Reply:
Ah, I see. I thought it meant using the validate fn.